File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
import sys
input = sys.stdin.readline

N = int(input())
arr = list(map(int, input().split()))
cnt = 0

for num in arr:
    temp = []
    for i in range(1, num):
        if num % i == 0:
            temp.append(i)
    if len(temp) == 1:
        cnt += 1
print(cnt)

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int[] arr = new int[n];
        int ans = 0;

        for (int i = 0; i < n; i++) {
            arr[i] = sc.nextInt();
        }

        for (int num : arr) {
            int cnt = 0;

            for (int j = 1; j <= num; j++) {
                if (num % j == 0) {
                    cnt++;
                }
            }

            if (cnt == 2) {
                ans++;
            }
        }

        System.out.println(ans);
    }
}
